code/game/gamemodes/clock_cult/clock_items/clockwork_slab.dm

```diff
@@ -327,57 +329,238 @@
+			that you choose an area in low-traffic part of the station (such as the auxilary base). Make sure to disconnect any cameras in the area beforehand.<br><br>"
+			dat += "Because of how complex being a servant is, it isn't possible to fit much information into this section. It's highly recommended that you read the <b>Components</b> \
+			and <b>Scripture</b> sections next. Not knowing how these two systems work will cripple both you and your fellows, and lead to a frustrating experience for everyone.<br><br>"
+			dat += "<font color=#BE8700 size=3>-=-=-=-=-=-</font>"
+		if("Terminology")
+			dat += "<font color=#BE8700 size=3>Common Servant Terminology</font><br>"
+			dat += "<i>This isn't intended to be read all at once; you are advised to treat it moreso as a glossary.</i><br><br>"
+			dat += "<font color=#BE8700 size=3>General</font><br>"
+			dat += "<font color=#BE8700><b>Servant:</b></font> A person or robot who serves Ratvar. You are one of these.<br>"
+			dat += "<font color=#BE8700><b>Cache:</b></font> A <i>tinkerer's cache</i>, which is a structure that stores and creates components.<br>"
+			dat += "<font color=#BE8700><b>CV:</b></font> Construction Value. All clockwork structures, floors, and walls increase this number.<br>"
+			dat += "<font color=#BE8700><b>Geis:</b></font> An important scripture used to make normal crew and robots into servants of Ratvar.<br>"
+			dat += "<font color=#BE8700><b>BE:</b></font> Belligerent Eye, a component type used in offensive scriptures.<br>"
+			dat += "<font color=#BE8700><b>VG:</b></font> Vanguard Cogwheel, a component type used in defensive scriptures.<br>"
+			dat += "<font color=#BE8700><b>GC:</b></font> Geis Capacitor, a component type used in mind-related scriptures.<br>"
+			dat += "<font color=#BE8700><b>RA:</b></font> Replicant Alloy, a component type used in construction scriptures.<br>"
+			dat += "<font color=#BE8700><b>HA:</b></font> Hierophant Ansible, a component type used in energy scriptures.<br>"
+			dat += "<font color=#BE8700><b>Ark:</b></font> The cult's win condition, a huge structure that needs to be defended.<br><br>"
+			dat += "<font color=#BE8700 size=3>Items</font><br>"
+			dat += "<font color=#BE8700><b>Slab:</b></font> A clockwork slab, a servant's most important tool. You're holding one! Keep it safe and hidden.<br>"
+			dat += "<font color=#BE8700><b>Visor:</b></font> A judicial visor, which is a pair of glasses that can stun everything in an area after a delay.<br>"
+			dat += "<font color=#BE8700><b>Wraith Specs:</b></font> Wraith spectacles, which provide true sight (x-ray, night vision) but damage the wearer's eyes.<br>"
+			dat += "<font color=#BE8700><b>Spear:</b></font> A Ratvarian spear, which is a very powerful melee weapon.<br>"
+			dat += "<font color=#BE8700><b>Proselytizer/Fabricator:</b></font> A replica fabricator, which converts objects into clockwork versions.<br><br>"
```

Comment:
I kept that as a thing because the slang from before the name change might still stick around. I intend/intended to remove it in a few weeks.